### DISCUSSION TITLE
is there a way to open the stream in other networks

### DISCUSSION THREAD

**Discussion by NQHuy1905** [bluenviron/mediamtx#1145] [Questions]
I could publish and open the stream by using devices in the same local network, but not with other networks

**Comment by aler9** [comment]
When readers and the server are placed in different LANs, in order to read a stream with the RTSP protocol you can use the TCP transport protocol as written in the README:
https://github.com/aler9/rtsp-simple-server#tcp-transport

[a comment by an automated account is not shown]
